Nodejs/Product/ProjectWizard/ImportWizard.cs

```diff
@@ -34,8 +35,8 @@ public sealed class NewProjectFromExistingWizard : IWizard {
 
         public void RunStarted(object automationObject, Dictionary<string, string> replacementsDictionary, WizardRunKind runKind, object[] customParams) {
             try {
-                Directory.Delete(replacementsDictionary["$destinationdirectory$"]);
-                Directory.Delete(replacementsDictionary["$solutiondirectory$"]);
+                // Regardless of whether or not an exception is thrown, created
+                // directories shall remain.
             } catch {
                 // If it fails (doesn't exist/contains files/read-only), let the directory stay.
             }
```

Comment:
Just delete this whole block along with any comments it contains since the body of the `try` is now empty.